src/Text/Text.tsx

```diff
@@ -66,9 +100,111 @@ Text.displayName = 'Text'
 
 const Container = styled(Box)<IText>(
   ({ align, color, cursor, typo }) => css`
-    /** PREDEFINED TYPOGRAPHY STYLES */
+    /** TYPOGRAPHY STYLES */
+
+    ${typo === 'hero-alternate' &&
+    css`
+      font-size: 40px;
```

Comment:
I don't think it's worth it. these will never change and I've ported them over from Figma double checking with Alistar that is all matching

Reply:
Not sure what you mean by more than a string, wouldn't this work?

```ts
const fontStyleMapping: Record<Typo, string> = {
  hero: `
      font-size: 32px;
      font-weight: ${theme.font.weight.medium};
      line-height: 40px;
      @media (min-width: 768px) {
        font-size: 40px;
        line-height: 44px;
      }
  `,
   ...
}
```

By having a mapping, we wouldn't need to maintain a separate array for the documentation as well since we can just infer it from the mapping:
```ts
const typos = Object.keys(fontStyleMapping)
```
Storybook would always be in sync with the codebase this way

Reply:
I don't think I get how that can be better than the way it's currently set up. Storybook is just rendering texts based on the `typo` how can it go out of sync currently? I think it's much easier to read if we have the actual styles in the css rather than in a mapping file

Reply:
oh ok now I got what you ment yeah I suppose we can do it that way